conda/recipes/dask-cuda/meta.yaml

```
@@ -29,9 +29,10 @@ requirements:
     - python
     - dask==2022.03.0
     - distributed==2022.03.0
-    - pynvml>=8.0.3
+    - pynvml>=11.0.0
     - numpy>=1.16.0
     - numba>=0.53.1
+    - click<8.1
```

Pinning click to 8.0.4 as suggested:

```suggestion
    - click==8.0.4
```